crates/widgets/src/widget/table_row.rs

```diff
@@ -182,6 +183,8 @@ where
             renderer,
             clipboard,
         );
+
+        event::Status::Ignored
```

Comment:
You should return the status from `self.content.on_event` since it needs to bubble up

Reply:
Thanks for the suggestion in the next commit i'm fixing it.